components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1709,14 +1740,22 @@ std::string AdsServiceImpl::LoadDataResourceAndDecompressIfNeeded(
   return data_resource;
 }
 
+// Custom Notifications and Message Center notifications use 2 different
+// types of notification.h
 void AdsServiceImpl::ShowNotification(
     const ads::AdNotificationInfo& ad_notification) {
-  auto notification = CreateAdNotification(ad_notification);
+  if (brave::IsDevOrCanaryBuild()) {
+    auto notification = CreateAdNotification(ad_notification);
 
-  std::unique_ptr<PlatformBridge> platform_bridge =
-      std::make_unique<PlatformBridge>(profile_);
+    std::unique_ptr<PlatformBridge> platform_bridge =
+        std::make_unique<PlatformBridge>(profile_);
 
-  platform_bridge->Display(profile_, notification);
+    platform_bridge->Display(profile_, notification);
+  } else {
+    auto notification = CreateMessageCenterNotification(ad_notification);
+    display_service_->Display(NotificationHandler::Type::BRAVE_ADS,
+        *notification, /*metadata=*/nullptr);
+  }
 
   StartNotificationTimeoutTimer(ad_notification.uuid);
```

Comment:
@yachtcaptain23 `StartNotificationTimeoutTimer` was not previously called for Android for native notifications, is this logic correct?

Reply:
Discussed with @yachtcaptain23 and resolved by not starting the timeout timer for release or beta builds on Android